articles/managed-operations/man-ref-customer-prereqs.md

```diff
@@ -0,0 +1,132 @@
+---
+title: Managed IT Operations prerequisites
+description: Customer prerequisites for Managed IT Operations
+services: managed-operations
+author: dbrennand
+reviewer: arapisarda
+lastreviewed: 21/09/2021
+toc_rootlink: Managed IT Operations
+toc_sub1:
+toc_sub2:
+toc_sub3:
+toc_sub4:
+toc_title: Managed IT Operations prerequisites
+toc_fullpath: Managed IT Operations/man-ref-customer-prereqs.md
+toc_mdlink: man-ref-customer-prereqs.md
+---
+
+# Managed IT Operations prerequisites
+
+## Overview
+
+Managed IT Operations helps to reduce the non-value add challenges of managing your compute assets, whilst increasing your overall level of trust and confidence in the true health of your entire multi-cloud estate. For more information, see the [*Managed IT Operations Service Definition*](man-sd-managed-it-ops.md).
+
+Before UKCloud can begin onboarding your virtual machines onto this service, your environment **must** meet the the following prerequisites.
+
+## Managed Monitoring as a Service
+
+with Managed Monitoring as a Service, UKCloud becomes your first line of support, 24 hours a day, 365 days a year, receiving and reacting to any abnormal events or alerts across your IT estate. For more information, see the [*Managed Monitoring as a Service Service Scope*](man-sco-monitoring.md).
+
+By default, UKCloud provides this service without the use of an agent by utilising SNMP traps. In situations where we need to collect non-SNMP events, we may opt to provide a dedicated Monitoring Agent, which can be deployed on to your virtual machines to provide a more advanced method of event collection.
+
+### Customer environment configuration
+
+Ensure the following ports listed below are permitted on both your permitter and operating system firewalls.
+
+#### Linux (Agent-based)
+
+- Nagios Remote Plugin Executor (NRPE) - Port 5666
+
+- Internet Control Message Protocol (ICMP)
+
```

Comment:
Under each section include "Any ports you would like monitoring. E.g. to monitor the state of the port for a web server running on port TCP/443. Port TCP/443 must be allowed through from the management bubble."